### ISSUE TITLE
This story is for Medium members

### ISSUE THREAD

**Issue by j3soon** [swapagarwal/mediumship#4]
If the new incognito window is created by the extension, sometimes Medium shows the following pop up:

![this story is for medium members](https://user-images.githubusercontent.com/20457146/39624643-a1850924-4fcc-11e8-806b-649f4b45c01e.png)

**Comment by j3soon** [comment]
Example [link](https://medium.com/bloomberg/this-court-battle-will-decide-who-will-make-a-fortune-from-gene-editing-tech-94f47fd39ce8).

**Comment by j3soon** [comment]
```
])}while(1);</x>{"success":true,"payload":{"value":[],"post":{"id":"94f47fd39ce8","versionId":"","creatorId":"3d76181076e6","creator":{"userId":"3d76181076e6","name":"Bloomberg","username":"bloomberg","createdAt":1472586420641,"lastPostCreatedAt":1522337665679,"imageId":"1*aFLLGsE20mi1mOuO8HH9ug.jpeg","backgroundImageId":"","bio":"Find more like this at bloomberg.com","twitterScreenName":"business","facebookAccountId":"","allowNotes":1,"mediumMemberAt":0,"isHouseAccount":true,"type":"User"},"homeCollectionId":"962fb1c4c47e","title":"","detectedLanguage":"en","latestVersion":"3e2f3d230957","latestPublishedVersion":"3e2f3d230957","hasUnpublishedEdits":false,"latestRev":141,"createdAt":1525189829000,"updatedAt":1525283318699,"acceptedAt":0,"firstPublishedAt":1525163458387,"latestPublishedAt":1525283317447,"experimentalCss":"","displayAuthor":"","coverless":true,"slug":"this-court-battle-will-decide-who-will-make-a-fortune-from-gene-editing-tech","translationSourcePostId":"","translationSourceCreatorId":"","isApprovedTranslation":false,"inResponseToPostId":"","inResponseToRemovedAt":0,"isTitleSynthesized":false,"allowResponses":true,"importedUrl":"","importedPublishedAt":1525163458387,"visibility":2,"uniqueSlug":"this-court-battle-will-decide-who-will-make-a-fortune-from-gene-editing-tech-94f47fd39ce8","license":0,"inResponseToMediaResourceId":"","canonicalUrl":"","approvedHomeCollectionId":"","newsletterId":"","webCanonicalUrl":"","mediumUrl":"","migrationId":"","notifyFollowers":false,"notifyTwitter":false,"isSponsored":false,"isRequestToPubDisabled":false,"notifyFacebook":false,"responseHiddenOnParentPostAt":0,"isSeries":false,"isSubscriptionLocked":true,"seriesLastAppendedAt":0,"audioVersionDurationSec":0,"sequenceId":"","isNsfw":false,"isEligibleForRevenue":false,"isBlockedFromHightower":false,"deletedAt":0,"lockedPostSource":3,"hightowerMinimumGuaranteeStartsAt":0,"hightowerMinimumGuaranteeEndsAt":0,"featureLockRequestAcceptedAt":0,"featureLockRequestMinimumGuaranteeAmount":0,"isElevate":false,"mongerRequestType":1,"type":"Post"},"highlights":[],"references":{"User":{"3d76181076e6":{"userId":"3d76181076e6","name":"Bloomberg","username":"bloomberg","createdAt":1472586420641,"lastPostCreatedAt":1522337665679,"imageId":"1*aFLLGsE20mi1mOuO8HH9ug.jpeg","backgroundImageId":"","bio":"Find more like this at bloomberg.com","twitterScreenName":"business","facebookAccountId":"","allowNotes":1,"mediumMemberAt":0,"isHouseAccount":true,"type":"User"}}}},"v":3,"b":"33614-f0cfeb6"}
```

The WebCanonicalUrl in https://medium.com/p/94f47fd39ce8/notes is empty: `"webCanonicalUrl":""`

However the link can be searched on google: https://www.bloomberg.com/news/articles/2018-04-29/berkeley-fights-harvard-mit-over-profits-from-gene-editing-tech

**Comment by swapagarwal** [comment]
Fixed in dd960d280997f67d98c5323f11e32979f9f1062a
